src/Microsoft.AspNet.SignalR.SqlServer/DbOperation.cs

```diff
@@ -137,27 +135,15 @@ private T Execute<T>(Func<DbCommand, T> commandFunc)
 #endif
```

Comment:
I cannot comment on the method declaration but I wonder if instead of `private T Execute<T>(Func<DbCommand, T> commandFunc)` shouldn't it be `private Task<T> Execute<T>(Func<DbCommand, Task<T>> commandFunc)`